src/components/clusters/ClusterDetailsMultiRegion/components/ClusterDetailsTop/components/RecommendedOperatorsAlert/ProductCardView.tsx

```diff
@@ -24,8 +26,13 @@ const ProductCardView = ({
   title,
   selectedCardTitle,
   learnMoreLink,
+  isHidden = false,
 }: ProductCardViewProps) => (
-  <div className={`${title ? `${title}-` : ''}product-card-view`}>
+  <div
+    className={`${title ? `${title}-` : ''}product-card-view`}
+    data-testid={`${title ? `${title}-` : ''}product-card-view`}
+    hidden={isHidden}
+  >
```

Comment:
This is maybe one of those rare cases where using html attributes like `hidden` is fine.
I would have probably used a PF utility class but in the end it does the same thing. Your solution is fine too.

Reply:
I haven't thought about that 😅 
Since it is a temporary solution until PF fixes the bug on their side, let's keep it as is